src/postgresql/test-unit/unittests.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+
```

Comment:
This should have a root namespace of `RapidCore.PostgreSql.UnitTests`

Reply:
fixed.